tests/acceptance/FormsListingCest.php

```diff
@@ -6,16 +6,20 @@
 
 class FormsListingCest {
   public function formsListing(\AcceptanceTester $i) {
+    $i->wantTo('Open forms listings page');
+    $formName = 'Test Form';
+    $newFormButton = '.mailpoet-button';
```

Comment:
This selector is too broad, if more buttons appear on the page, the test will be broken. One of the buttons already has `data-automation-id="create_new_form"`, we can add the same id to another button, or, to not mix them up, use the `create_new_form_2` ID and select both with `[data-automation-id^="create_new_form"]` (`^=` means "begins with").